content/en/users/workload-manager/_index.md

```diff
@@ -4,9 +4,460 @@ linkTitle: "Workload Manager"
+
+The EGI Workload Manager:
+Maximizes usage efficiency by choosing appropriately computing and storage
+resources on real-time.
+Provides a large-scale distributed environment to manage and handle data
+storage, datamovement, accessing and processing.
+Handles job submission and workload distribution in a transparent way.
+Improves the general job throughput compared with native management of EGI
+grid or cloud computing resources.
+Offers pilot-based task scheduling method, that submits pilot jobs to
+resources to check the execution environment before to start the user's jobs.
+From a technical standpoint, the user's job description is delivered to the
+pilot, which prepares its execution environment and executes the user application.
+The pilot-based scheduling feature solves many problems of using
+heterogeneity and unstable distributed computing resources.
+Includes easy extensions to customize the environment checks to address the
+needs of a particular community.
+Users can choose appropriately computing and storage resources maximising their
+usage efficiency for particular user requirements.
+Handles different storage supporting both cloud and grid capacity.
+Provides a user-friendly interface that allows users to choose among different
+DIRAC services.
+
+## Target User Groups
+
+The service suits for the established Virtual Organization communities, long
+tail of users, SMEs and Industry
+EGI and EGI Federation participants
+Research communities
+
+## Architecture
+
+The EGI Workload Manager service (DIRAC4EGI) is a cluster of DIRAC services
+running on EGI resources (HTC, CLOUD, HPC) supporting multi-VO.  All the DIRAC
+services are at or above TRL8.  The main service components include: Workload
+Management System (WMS) architecture is composed of multiple loosely coupled
+components working together in a collaborative manner with the help of a common
+Configuration Services ensuring reliable service discovery functionality.  The
+modular architecture allows to easily incorporate new types of computing
+resources as well as new task scheduling algorithms in response to evolving user
+requirements.  DIRAC services can run on multiple geographically distributed
+servers which increases the overall reliability and excellent scalability
+properties.  REST server providing language neutral interface to DIRAC service.
+Web portal provides simple and intuitive access to most of the DIRAC
+functionalities including management of computing tasks and distributed data.
+It also has a modular architecture designed specifically to allow easy extension
+for the needs of particular applications.
+
+![Fig. 1 - The DIRAC Web portal](./figure_1.png)
+
+Fig. 1 - The DIRAC Web portal
+
+## How to access the EGI Workload Manager service
+
+There are several options to access the service:
+
+1. Members of a scientific community whose resources pool is already configured
+   in the DIRAC4EGI instance > can use the DIRAC4EGI web portal to access the
```

Comment:
DIRAC4EGI is a deprecated name and shouldn't anymore be used.

Reply:
1e375a0a6b3609ef95c40d906e3a8fd61b174cd5